nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc

```diff
@@ -225,6 +227,7 @@ void NNTrainer::InputTensorsInfo::getSample(float **input, float **label,
     is_data_wait_locked = TRUE;
     data_wait.wait(lock);
     ml_logd("unlocked, get data");
+    is_data_wait_locked = FALSE;
```

Comment:
is_data_wait_locked is not protected. are you sure you can use it like this?

Reply:
PR is changed, Thank you.